### PR TITLE
update README.md

### PR DESCRIPTION
**Pull request by kimyoungjae96**
I drafted the README. Please check it out.
and can you give me the permission to update wiki?
I want to update the wiki. ex) https://github.com/kimyoungjae96/ko.mobx.js.org/wiki

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,172 +14,36 @@ _Simple, scalable state management._
...
+PR을 올릴 시 체크리스트를 꼭 확인해주세요.
+PR에는 이슈를 확인할 수 있게 의미 있는 제목을 기재해주세요.
+#### 예시
+- [문서번역] installation.md
+- [오역수정] lazy-observables.md
+
+### 로컬 서버 세팅 방법
+1. 개인 저장소에 포크 받은 후, 포크된 저장소를 클론합니다.
+```angular2html
+git clone https://github.com/나의GitHub유저네임/ko.mobx.js.org
```
Comment (iChenLei) [inline README.md:43]: Invalid github link?
> https://github.com/나의GitHub유저네임/ko.mobx.js.org

Comment (kimyoungjae96) [reply README.md:43]: That is just an example. ex) https://github.com/username/ko.mobx.js.org, https://github.com/kimyoungjae96/ko.mobx.js.org 

You can think of it as below.
나의GitHub유저네임 => username

**Comment by iChenLei** [comment]
> can you give me the permission to update wiki?
I am not super admin of mobxjs org, so I can't give you permission ( only mobxjs team member can edit it currently

> I want to update the wiki. ex) https://github.com/kimyoungjae96/ko.mobx.js.org/wiki
Maybe you can look this question from stackoverflow https://stackoverflow.com/questions/10642928/how-to-pull-request-a-wiki-page-on-github, I update wiki manually now.

File: README.md (hunk 1 above)
Comment (iChenLei) [reply README.md:43]: ooops, my fault, sorry

**Comment by kimyoungjae96** [comment]
> > can you give me the permission to update wiki?
> 
> I am not super admin of mobxjs org, so I can't give you permission ( only mobxjs team member can edit it currently
> 
> > I want to update the wiki. ex) https://github.com/kimyoungjae96/ko.mobx.js.org/wiki
> 
> Maybe you can look this question from stackoverflow https://stackoverflow.com/questions/10642928/how-to-pull-request-a-wiki-page-on-github, I update wiki manually now.

I think you're having a hard time because of me. I'm sorry, and thank you so much.

**Comment by iChenLei** [comment]
> I think you're having a hard time because of me. I'm sorry, and thank you so much.
 I am proud of any OSS work, continue your great work, @ me if you need any help.

**Comment by kimyoungjae96** [comment]
I changed my wiki address to mobx. Tell me if there's anything more to do.

**Review by iChenLei — APPROVED** [review]
Ready to merge ?

**Comment by kimyoungjae96** [comment]
> Ready to merge ?

yes, please.

**Pull request merged**